tools/benchmark/rs-benchmark.cpp

```diff
@@ -206,6 +207,10 @@ class gl_blocks : public suite
 
 int main(int argc, char** argv) try
 {
+    std::string serial;
+    if (!device_with_streams({ RS2_STREAM_COLOR,RS2_STREAM_DEPTH }, serial))
```

Comment:
make the demo utilize IR stream if color is not present.
test flow-
1. Depth stream - must
2 . Color or Infrared stream (2nd priority) - must